lms/services/exceptions.py

```diff
@@ -125,12 +121,7 @@ def raise_from(cls, cause, request, response):
         exception_class = cls._exception_class(response)
 
         raise exception_class(
-            message="Calling the Canvas API failed",
-            request=request,
-            response=response,
-            extra_details={
-                "validation_errors": getattr(cause, "messages", None),
-            },
```

Comment:
This is some Canvas-specific code that causes the validation errors (but not the request or response) to be sent to the frontend if validating a Canvas API response fails. I'm working on a PR that will send the request, response and validation errors to the frontend, Sentry and Papertrail for all external request validation errors (https://github.com/hypothesis/lms/pull/3324) and that will obsolete this Canvas-specific code